### ISSUE TITLE
Status only works once MD enabled

### ISSUE THREAD

**Issue by patschi** [jacopotediosi/OctoPrint-Telegram#194]
I'm having a very strange issue:. For whatever reason the `/status` command only works once - as seen here:
![image](https://user-images.githubusercontent.com/2029878/62726341-508ddf80-ba17-11e9-92d8-feead39df167.png)

The bot appears to work, however not reacting to the status command anymore.

It is reproducible following way:
1. Go to "Zchange"
2. Set "Command message" to "Separated"
3. Set "Markup selection" to "MD" (Markdown)
4. Save settings
5. Now `/status` only works once.
6. The even funnier thing: When you go to the same settings and you change back "Command message" to "Combined", it still doesn't work once and there's no message appearing anymore.

tl;dr: Something when the Markdown parser or so seems to be broken.

**Comment by giloser** [comment]
Hi 


did you have the gif activate? it can take some time before he respond, we will add a temp message to let know that the gif is in creation .

**Comment by patschi** [comment]
No, I haven't enabled the new GIF feature on that message. As written earlier, once I change "Markup selection" to something else than "MD", it works instantly just fine. (also without GIF)

Not related to the issue: I tried using `/gif` before once, but then nearly killed the printing job and the whole Raspberry Pi went OOM. I believe generating a GIF from multiple 3240x2464 photos simply takes too much resources from that little thing.

**Comment by giloser** [comment]
Hi,
as I have made an error and the gif is activate by default for the Z change message can you check your the gif features is really not activate for this message?

thanks

**Comment by patschi** [comment]
In the main settings above, it is:
![image](https://user-images.githubusercontent.com/2029878/62769546-7d82d680-ba99-11e9-9e31-3c3090b48b45.png)

But next to the Zchange it isn't:
![image](https://user-images.githubusercontent.com/2029878/62769605-9e4b2c00-ba99-11e9-8a4d-e65afbd3126e.png)

**Comment by jacopotediosi** [comment]
The user did not attach their logs. However, it is likely something related to webcam handling or message formatting, which should have been fixed by recent commits in the `devel` branch (e.g., 1dd2fd61b0eb8edec72fd2e4c4970fedd24da8cf).
